onnx_tf/handlers/backend/dequantize_linear.py

```diff
@@ -0,0 +1,39 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+
+
+@onnx_op("DequantizeLinear")
+class DequantizeLinear(BackendHandler):
+
+  @classmethod
+  def args_check(cls, node, **kwargs):
+    x = kwargs["tensor_dict"][node.inputs[0]]
+    x_scale = kwargs["tensor_dict"][node.inputs[1]]
+    x_zero_point = tf.zeros((), x.dtype)
+    if node.inputs[2] in kwargs["tensor_dict"]:
+      x_zero_point = kwargs["tensor_dict"][node.inputs[2]]
+    if x_scale.shape != x_zero_point.shape:
+      raise ValueError("DequantizeLinear x_scale(shape=" + str(x_scale.shape) +
+                       ") and x_zero_point(shape=" + str(x_zero_point.shape) +
+                       ") must be in the same shape")
+    if x_zero_point.dtype != x.dtype:
+      raise ValueError("DequantizeLinear x_zero_point(" +
+                       str(x_zero_point.dtype) + ") and x(" + str(x.dtype) +
+                       ") must be in the same dtype")
+
+  @classmethod
+  def version_10(cls, node, **kwargs):
+    x = kwargs["tensor_dict"][node.inputs[0]]
+    x_scale = kwargs["tensor_dict"][node.inputs[1]]
+    x_zero_point = tf.zeros((), x.dtype)
+    if node.inputs[2] in kwargs["tensor_dict"] and x.dtype != tf.int32:
```

Comment:
Please explain why checking x.dtype != tf.int32.

Reply:
In ONNX description, it state that "In the case of dequantizing int32, there's no zero point (zero point is supposed to be 0)."

> Please explain why checking x.dtype != tf.int32.
In ONNX description, it state that "In the case of dequantizing int32, there's no zero point (zero point is supposed to be 0)."

Reply:
Got it. Thanks.